code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -13,20 +13,29 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 	var/r_ears2 = 30	// Ear extra color.
 	var/g_ears2 = 30	// Ear extra color
 	var/b_ears2 = 30	// Ear extra color
+	var/r_ears3 = 30	// Ear tertiary color.
+	var/g_ears3 = 30	// Ear tertiary color
+	var/b_ears3 = 30	// Ear tertiary color

```

Comment:
This is getting to a point where you should probably just be serializing/deserializing a colour matrix :P (this is a joke, not a change request)